packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -357,11 +357,11 @@ class AOTSnapshotter {
   /// to find.
   /// Important: external performance tracking tools expect format of this
   /// output to be stable.
-  Future<T> _timedStep<T>(String marker, FutureOr<T> Function() action) async {
+  Future<T> _timedStep<T>(String marker, String metric, FutureOr<T> Function() action) async {
     final Stopwatch sw = Stopwatch()..start();
     final T value = await action();
     if (reportTimings) {
-      printStatus('$marker(RunTime): ${sw.elapsedMilliseconds} ms.');
+      printStatus('$marker($metric): ${sw.elapsedMilliseconds} ms.');
```

Comment:
Maybe just replace this with `'$marker:'` and then use it like `_timedStep('frontend(CompileTime)', ...)`.

Reply:
Done.